src/main/java/bio/terra/common/exception/AbstractGlobalExceptionHandler.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.common.exception;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+import java.util.List;
+
+/**
+ * This module provides an abstract as a top-level exception handler for controllers. All exceptions that rise
+ * through the controllers are caught in this handler. It converts the exceptions into standard
```

Comment:
"`All exceptions that rise through the controllers are caught in this handler.`" That sounds like it happens magically, but I think that's aspirational. Let's change the phrasing so that it's clear what this base class does, vs what should be done by subclasses.